### ISSUE TITLE
Errors with vignette during module 3.2

### ISSUE THREAD

**Issue by ressy** [scharch/zap#3]
I'm stepping through the example dataset both using runVignette.sh in Docker and running the commands myself as an exercise, and all goes well up to the longitudinal part.  I'm having trouble when it reaches the igphyml step.  It looks like there are a few different things happening but I'm not sure if they're related.

First in phylogeny/3.2-run_IgPhyML.py line 381, it checks `arugments[--natives]`, finds it is not None, and continues, but for some reason that argument is the literal string "False":

```
loading sequence info from /SONAR/germDB/IgHV.fa...
loading sequence info from False...
Traceback (most recent call last):
  File "/SONAR/phylogeny/3.2-run_IgPhyML.py", line 380, in <module>
    natives = load_fastas( arguments['--natives'] )
  File "/SONAR/__init__.py", line 277, in load_fastas
    reader, result = SeqIO.parse(open(f, "r"), "fasta"), dict()
FileNotFoundError: [Errno 2] No such file or directory: 'False'
```
I got past that by replacing this:

    if arguments['--natives'] is not None:

With this, assuming that it *should* be equivalent to None in this case:

    if arguments['--natives'] is not None and arguments['--natives'] != "False":

It continues, but later on this happens:

```
stats:
Error running 'perl -I /SONAR/third-party /SONAR/third-party/ancReconstructHLP17.pl /host/home/cap256-longitudinal/work/phylo/ar.config':
Use of uninitialized value $statsfile in concatenation (.) or string at /SONAR/third-party/ancReconstructHLP17.pl line 131.
kappa needs to be specified at /SONAR/third-party/ancReconstructHLP17.pl line 269.
Exit code 255
```

Looks like that's inside main() around line 300 where it sets up that config file and passes it to the perl script:

                with open("%s/ar.config"%prj_tree.phylo, "w") as handle:
                        handle.write( "length\t%d\n" % (align_len/3) )
                        handle.write( "rooted\t1\noutdir\t%s\n" % prj_tree.phylo )
                        handle.write( "seqfile\t%s/infile\n" % prj_tree.phylo )
                        handle.write( "rootid\t%s\n" % germ_id )
                        handle.write( "igphyml\t%s/%s\n" % (SCRIPT_FOLDER, "third-party") )
                        handle.write( "stats\t%s/infile_igphyml_stats.txt_hlp17\n" % prj_tree.phylo )
                        handle.write( "tree\t%s/infile_igphyml_tree.txt_hlp17\n" % prj_tree.phylo )
                        handle.write( "ambigfile\t%s/ambigfile.txt\n" % prj_tree.phylo )
                        handle.write( "stem\t%s\n" % prj_name )

                        s = subprocess.Popen( ["perl", "-I", "%s/third-party"%SCRIPT_FOLDER, reconstruct, "%s/ar.config"%prj_tree.phylo], universal_newlines=True, stderr=subprocess.PIPE )
                        o,e = s.communicate()

It looks like kappa can come from the config or stats file, and I *do* have a infile_igphyml_stats.txt_hlp17 file with the line ". Transition/transversion ratio:        1.74581" that looks like it should do the trick but it must not be getting set as kappa in the Perl script for some reason.

Any ideas?  This happens when I set up my own environment with dependencies and run the steps manually, or if I run from scratch with Docker using runVignette.sh.  Maybe something has changed since version 4.0 and I should give that a try, too; I just realized the Docker setup goes with the latest master branch so I haven't tried using the latest stable release instead for any of this.  Thanks!

**Comment by scharch** [comment]
Turns out that there were some stray tabs in the docstring that were messing up the `docopt` parsing. This has now been fixed with commit `447b066`.

**Comment by scharch** [comment]
Right, you reported a second bug, too. That has now been fixed, as well, with commit `b274215`.

**Comment by ressy** [comment]
That was quick, thanks!  I've pulled in the updates in my docker container and I'm re-running 3.2 now.  So far so good.
